src/api/spec/models/update_notification_events_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 RSpec.describe UpdateNotificationEvents, type: :model do
   let!(:admin) { create(:admin_user, login: 'Admin') }
+  let!(:project_with_package) { create(:project_with_package, name: 'project_1', package_name: 'multibuild') }
```

Comment:
after creating a relationship with a package on `Event::Base`, passing nonexistent package and project could be a problem

Reply:
We are gonna remove the relationship between `Event` and `Package` so this won't be a problem anymore.